### PR TITLE
Go Client code samples corrected. [API-1360]

### PR DESCRIPTION
**Pull request by mehmettokgoz**

### CODE REVIEW

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ Java::
 --
 [source,java]
 ----
-Map<Integer, String> countries = hzClient.getMap( "countries" );
+Map<Integer, String> countries = hzClient.getMap("countries" );
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:19]: I think that space was intentional. Wdyt?

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 2]
```diff
@@ -83,12 +83,13 @@ Go::
...
 [source,go]
 ----
-countries, _ := hzClient.GetMap("countries")
-countries.Put( 1, "Japan" );
-countries.Put( 2, "France" );
-countries.Put( 3, "USA" );
-countries.Put( 4, "Turkey" );
-countries.Put( 5, "Ukraine" );
+ctx := context.Background()
+countries, _ := hzClient.GetMap(ctx, "countries")
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:87]: Can you add a comment about errors

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 3]
```diff
@@ -173,9 +174,10 @@ Go::
 -- 
 [source,go]
 ----
-countries.Put( 1, "Korea" ); <1>
-countries.Put( 2, "France", "Spain" ); <2>
-countries.Put( 3, "United States", "Canada" ); <3>
+ctx := context.Background()
+countries.Replace(ctx, 1, "Korea") <1>
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:178]:
```suggestion
result, _ := countries.Replace(ctx, 1, "Korea") <1>
```
How about prepending these lines with return value and a ignored error? And also include a comment that errors are ignored for brevity?

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 4]
```diff
@@ -278,14 +280,15 @@ Go::
 [source,go]
 ----
 // error checking is omitted for brevity
-distributedMap, _ := hzClient.GetMap("map")
+ctx := context.Background()
+distributedMap, _ := hzClient.GetMap(ctx, "map")
 entries := []types.Entry{
-	types.NewEntry(1, "John"),
+    types.NewEntry(1, "John"),
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:286]: I think we need to fix the indentation for the other lines as well, or revert this

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 5]
```diff
@@ -278,14 +280,15 @@ Go::
...
 	types.NewEntry(2, "Jack"),
 	types.NewEntry(3, "David"),
 }
-distributedMap.PutAll(entries)
-size, _ := distributedMap.Size()
+distributedMap.PutAll(ctx, entries...)
+size, _ := distributedMap.Size(ctx)
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:291]: Not this line, but can you remove 2-3 lines below where it says
`
// error checking is omitted for brevity
distributedMap, _ := hzClient.GetMap("map")
`

File: docs/modules/data-structures/pages/updating-map-entries.adoc [hunk 6]
```diff
@@ -408,8 +411,9 @@ Go::
 [source,go]
 ----
 // error checking is omitted for brevity
-studentMap, _ := hzClient.GetMap("studentMap")
-studentMap.RemoveAll(predicate.Equal("GradYear", 2020))
+ctx := context.Background()
```
Comment (utku-caglayan) [inline docs/modules/data-structures/pages/updating-map-entries.adoc:414]: Even the Java sample is not correct for this section. How about fixing them all?

**Review by utku-caglayan — COMMENTED** [review]
Thanks, I posted few comments

File: docs/modules/data-structures/pages/updating-map-entries.adoc (hunk 6 above)
Comment (yuce) [reply docs/modules/data-structures/pages/updating-map-entries.adoc:414]: I think let's fix the Go bits and notify the documentation team about code for other clients.

File: docs/modules/data-structures/pages/updating-map-entries.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/data-structures/pages/updating-map-entries.adoc:19]: Yes, let's not touch the non-Go client bits in the doc.

**Review by yuce — APPROVED** [review]

**Review by utku-caglayan — APPROVED** [review]

**Pull request merged**